core/trino-main/src/main/java/io/trino/operator/scalar/MathFunctions.java

```diff
@@ -57,11 +49,7 @@
 import static io.trino.spi.type.Decimals.longTenToNth;
 import static io.trino.spi.type.Decimals.overflows;
 import static io.trino.spi.type.DoubleType.DOUBLE;
-import static io.trino.spi.type.Int128Math.add;
-import static io.trino.spi.type.Int128Math.negate;
-import static io.trino.spi.type.Int128Math.rescale;
-import static io.trino.spi.type.Int128Math.rescaleTruncate;
-import static io.trino.spi.type.Int128Math.subtract;
+import static io.trino.spi.type.Int128Math.*;
```

Comment:
Instead of having import all - can we import the class on-demand ?

Reply:
Reverted